js-pkg/client/src/keystore.ts

```diff
@@ -0,0 +1,29 @@
+import { exportKey, generateEncryptionKey, importKey } from './encryption'
+
+const COOKIE_NAME = 'YSWEET_OFFLINE_KEY'
+
+function getCookie(name: string): string | null {
+  const value = `; ${document.cookie}`
+  const parts = value.split(`; ${name}=`)
+  if (parts.length === 2) return parts.pop()?.split(';').shift() || null
+  return null
+}
+
+function setCookie(name: string, value: string) {
+  document.cookie = `${name}=${value};path=/;expires=Fri, 31 Dec 9999 23:59:59 GMT;secure`
```

Comment:
- WebCrypto does not have a mechanism for key management.
- Cookies are more secure in modern browsers than IndexedDB, because IndexedDB is stored in plaintext while cookies are encrypted.